### PR TITLE
Feature/39/adding new transaction to db

### PR DESCRIPTION
**Pull request by axseinga**
1. moved baseUrl of transaction db to .env file;
2. add service API to add new transaction
3. fix issues with validation

### CODE REVIEW

**Review by hannakozak — APPROVED** [review]

File: src/components/common/PageNumberButton/PageNumberButton.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <StyledListItem onClick={() => paginate(pageNumber)}>
+      <StyledListLink href="#">{pageNumber}</StyledListLink>
+    </StyledListItem>
+  );
+};
+
+PageNumberButton.propTypes = {
+  pageNumber: PropTypes.number,
+  paginate: PropTypes.func,
+};
```
Comment (m-ossolinski) [inline src/components/common/PageNumberButton/PageNumberButton.js:15]: are you sure that those files are needed and related to your task?

File: src/components/common/PageNumberButton/PageNumberButton.styled.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  width: 4.5rem;
+  line-height: 4.5rem;
+  border-radius: 50%;
+  background-color: #fff;
+  margin-right: 5px;
+  font-size: 15px;
+  text-decoration: none;
+  text-align: center;
+  color: #131313;
+`;
```
Comment (m-ossolinski) [inline src/components/common/PageNumberButton/PageNumberButton.styled.js:16]: ^

File: src/components/common/PaginateButton/PaginateButton.style..js [hunk 3]
```diff
@@ -7,10 +7,19 @@ export const StyledListLink = styled.a`
...
+  color: #131313;
+=======
   margin-right: 5px;
   font-size: 15px;
   background-color: ${(props) => props.bg};
   text-decoration: none;
   text-align: center;
   color: ${(props) => props.textColor};
+>>>>>>> develop
 `;
```
Comment (m-ossolinski) [inline src/components/common/PaginateButton/PaginateButton.style..js:25]: ^

**Review by m-ossolinski — CHANGES_REQUESTED** [review]

**Comment by axseinga** [comment]
new branch ->  feature/39/adding-new-transaction-to-db-v2

**Pull request closed without merging**